src/test/resources/com/microsoft/azure/vmagent/test/jcasc/expectedBasic.yaml

```diff
@@ -13,6 +13,7 @@
       doNotUseMachineIfInitFails: true
       executeInitScriptAsRoot: true
       imageReference:
+        galleryImageSpecialized: false
```

Comment:
this shouldn't need to be changed if you switch it to a setter
```suggestion
```

Reply:
Yes, you are right. I was wondering why it is expected in Basic configuration.